sdk/component/internal/storer/local/sqlite/export_test.go

```diff
@@ -0,0 +1,27 @@
+package sqlite
+
+import (
+	"fmt"
+)
+
+// CreateTable is used to create a table in testing settings.
+func (m *manager) CreateTable() error {
```

Comment:
To be removed when we have a better migrations story post POC and discussion with the team about data structure.